src/util/BrowserFeatures.js

```diff
@@ -58,6 +58,14 @@ define(['underscore'], function (_) {
     return /(msie|trident)/i.test(navigator.userAgent);
   };
 
+  fn.isFirefox = function () {
+    return navigator.userAgent.toLowerCase().indexOf('firefox') > -1;
+  };
+
+  fn.isChrome = function () {
+    return /Chrome/.test(navigator.userAgent) && /Google Inc/.test(navigator.vendor);
+  };
```

Comment:
I have different mocks for the Firefox and other browsers.
In Chrome feature is available from the versions 38.
In Firefox feature is not available without the specific extension.
So, for Firefox I should check the browser, not feature.

I can refactor the code and remove the isChrome function.